contrib/download-frozen-image-v2.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
```

Comment:
@vdemeester I can take care of it if you'd like -- then I can ping you to review it :smile: :+1: :heart: